src/praisonai-agents/praisonaiagents/agent/agent.py

```
@@ -592,6 +592,7 @@ def __init__(
         runtime: Optional[Union[str, Dict[str, Any], 'AgentRuntimeConfig']] = None,  # Model-scoped runtime configuration
         interrupt_controller: Optional['InterruptController'] = None,  # G2: Cooperative cancellation
         tool_search: Optional[Union[bool, str, Dict[str, Any], 'ToolSearchConfig']] = False,  # Progressive tool disclosure
+        tool_output: Optional[Union[bool, Dict[str, Any], 'ToolOutputConfig']] = None,  # Tool output handling and artifact storage
```

Ruff (0.15.17)

[error] Undefined name `ToolOutputConfig`

(F821)